core/src/main/scala/org/broadinstitute/dsde/rawls/billing/BillingProfileManagerClientProvider.scala

```diff
@@ -32,7 +32,7 @@ class HttpBillingProfileManagerClientProvider(baseBpmUrl: Option[String]) extend
     val client: ApiClient = new ApiClient()
 
     val clientConfig = new ClientConfig()
-    clientConfig.connectorProvider(new JdkConnectorProvider)
+    clientConfig.connectorProvider(new JavaNetHttpConnectorProvider())
```

Comment:
Can we remove `jersey-jdk-connector` as a dependency? Though I'm not finding it, so perhaps it is inherited from somewhere else.

Reply:
Yeah, it's pulled in via WSM so there's no direct dependency to remove